source/frontend/src/features/properties/map/acquisition/detail/AcquistionOwnersSummaryContainer.tsx

```diff
@@ -0,0 +1,106 @@
+import { Api_AcquisitionFile, Api_AcquisitionFileOwner } from 'models/api/AcquisitionFile';
+import { Api_Address } from 'models/api/Address';
+import React from 'react';
+import { useCallback, useEffect, useState } from 'react';
+
+import { useAcquisitionProvider } from '../hooks/useAcquisitionProvider';
+import { DetailAcquistionFileOwner } from './models';
+
+export interface IAcquisitionOwnersContainerProps {
+  acquisitionFile?: Api_AcquisitionFile;
+  View: React.FC<IAcquisitionOwnersSummaryViewProps>;
+}
+
+export interface IAcquisitionOwnersSummaryViewProps {
+  ownersList?: DetailAcquistionFileOwner[];
+  isLoading: boolean;
+}
+
+const AcquisitionOwnersSummaryContainer: React.FunctionComponent<
```

Comment:
Seem to have some coverage issues with this file, are more tests for this coming?

Reply:
There's not much logic to put more tests in this.